open-amp/lib/remoteproc/remoteproc.c

```diff
@@ -299,10 +299,12 @@ void remoteproc_init_mem(struct remoteproc_mem *mem, const char *name,
 {
 	if (!mem || !io || size == 0)
 		return;
-	if (name)
-		strncpy(mem->name, name, sizeof(mem->name));
-	else
+	if (name) {
+		strncpy(mem->name, name, sizeof(mem->name) - 1);
+		mem->name[sizeof(mem->name) - 1] = 0;
```

Comment:
minor nit, `0` or `'\0'` :)

Reply:
I personally prefer `'\0'`, but here there was already an instance of `= 0`, so that's why. :slightly_smiling_face: